src/components/DayPickerRangeController.jsx

```diff
@@ -160,6 +160,8 @@ export default class DayPickerRangeController extends React.Component {
     if (focusedInput !== END_DATE) return false;
 
     if (startDate) {
+      if (minimumNights === 1) return isSameDay(day, startDate);
```

Comment:
This doesn't seem to be an issue that would just be relegated to `minimumNights` not equaling 1... It seems like it would cause problems for any minimumNights value, no?